pkg/services/sqlstore/sqlstore.go

```diff
@@ -497,7 +501,15 @@ func initTestDB(migration registry.DatabaseMigrator, opts ...InitTestDBOpt) (*SQ
 
 		// set test db config
 		cfg := setting.NewCfg()
-		cfg.IsFeatureToggleEnabled = func(key string) bool { return false }
+		cfg.IsFeatureToggleEnabled = func(requestedFeature string) bool {
```

Comment:
we are reusing the same store in different tests so if we have a migration hidden behind a feature flag we have to enable it beforehand. not sure if that's the best way of doing it @FZambia @ryantxu